cypress.config.js

```diff
@@ -29,6 +29,17 @@ module.exports = defineConfig({
           return null;
         },
       });
+
+      on('task', {
+        generateTestData() {
+          return {
+            edited_username: faker.name.firstName(),
+            edited_bio: faker.lorem.word(5),
+            edited_email: faker.internet.email(),
+            edited_password:  faker.internet.password({ length: 8 }),
```

Comment:
Use 'faker.internet.password(8)' instead of 'faker.internet.password({ length: 8 })' to generate a password with 8 characters.

Reply:
fixed